README.md

```diff
@@ -32,7 +32,7 @@ Pkg.add("DocsScraper")
 
 ## Building the Index
 ```julia
-index = make_knowledge_packs(; single_urls=["https://docs.sciml.ai/Overview/stable/"], index_name="sciml", embedding_size=1024)
+index = make_knowledge_packs(["https://docs.sciml.ai/Overview/stable/"]; index_name="sciml", embedding_size=1024, bool_embeddings=true)
```

Comment:
Looks good! I would suggest making them consistent, eg, `embedding_size`, embedding_bool` (it's easier for users to remember)

Reply:
Not sure how hard it is to change it in the user-facing functions, but the kwarg name in AIHelpMe is `embedding_dimension` (not size)